varsim_multi.py

```diff
@@ -207,6 +241,27 @@ def varsim_multi(reference,
                                 required=False)
     rand_dgv_group.add_argument("--sv_dgv", metavar="DGV_FILE", help="DGV file containing structural variants",
                                 required=False)
+    rand_dgv_group.add_argument("--sv_prop_het", metavar="FLOAT", help="Proportion of heterozygous structural variants",
+                                default=0.6,
+                                type=float)
+    dwgsim_group = main_parser.add_argument_group("DWGSIM options")
+    dwgsim_group.add_argument("--dwgsim_start_e", metavar="first_base_error_rate", help="Error rate on the first base",
```

Comment:
Why did you add simulator-specific options?

Reply:
if we don't provide simulator-specific options, then users need to figure out what they need to use for each simulator. e.g., `art` requires `-l` for read length, but the user in #148 didn't know about this. By providing these simulator-specific options (along with reasonable default values), varsim might be more user-friendly.

please see here for previous simulator option: https://github.com/bioinform/varsim/pull/149/files#diff-163a888df028d8bfa31a54c7682b7a12R312

Thanks.

Reply:
That would make it messy to support more reads simulators. I wanted to avoid having to parse simulator-specific options only to pass them to the simulator. I'd suggest not adding these simulator-specific options and instead have simulator-specific parsers which later parse `args.simulator_options` and do error-checks.